### ISSUE TITLE
Feature Suggestion: an indication of which player is which when spectating a match.

### ISSUE THREAD

**Issue by Createor** [SirZach/bbbbbbbbbbbbbbb#212]
When spectating it's difficult to tell which player is which. Is it top vs bottom, or bottom vs top. The UI could make this clearer, perhaps through the use of color coding and/or labeling?

**Comment by SirZach** [comment]
I agree with this. The UI is going to have to get a little more cluttered and clever to fully support what's necessary.
